test/performance/heterogeneous/dgl/han_main.py

```diff
@@ -0,0 +1,311 @@
+"""
+Performance check of AutoGL model + DGL (trainer + dataset)
+"""
+import os
+import datetime
+import errno
+import numpy as np
+from tqdm import tqdm
+import dgl
+os.environ["AUTOGL_BACKEND"] = "dgl"
+import sys
+import os.path as osp
+sys.path.append("../../../../")
```

Comment:
these lines should be resolved if you install the local library through `pip install -e .`